WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PersonsController < ApplicationController
+  def index
+    @regions = { 'Continent' => Continent.all.map { |continent| [continent.name, continent.id] },
```

Comment:
OK, if it's used in more places it really needs to be factored out.

I'd probably put the whole thing in `ApplicationHelper.region_opt_groups`. I'd let the data come from the classes. In part because @-caching in helpers seems icky... Like so:

    class Continent < ActiveRecord::Base
      self.table_name = "Continents"

      def self.as_opt_group
          @opt_group ||= self.all.map { |continent| [continent.name, continent.id] }
      end
    end